src/FroshDevelopmentHelper.php

```diff
@@ -21,4 +23,21 @@ public function build(ContainerBuilder $container): void
         $container->addCompilerPass(new BuildEntityDefinitionNamesCompilerPass());
         parent::build($container);
     }
+
+    public function install(InstallContext $installContext): void
```

Comment:
Okay, thanks for clarifying! So you install the dependencies in the CI and distribute it with the complete vendor folder, is that correct?

Reply:
yep. But I would do it with composer 1 currently. Composer 2 generates an InstalledVersions this will colidate with some other plugins or core itself

Reply:
I just tried to install composer locally in a project and it works:
```
$ composer init -n --name rafaelkr/test --require composer/composer:^2
$ composer install
$ vendor/bin/composer -V
Composer 2.0.11 2021-02-24 14:57:23
```

Maybe it could be shipped with Shopware itself?
Or will it always be preferred to ship plugins with the vendor folder?

Reply:
> Composer 2 generates an InstalledVersions this will colidate with some other plugins or core itself

So it's currently recommended to use Composer 1 with Shopware 6 in all cases?

Reply:
It should be fine when we delete that file https://github.com/shyim/php-dump-client/blob/master/build-prefix.php#L6